src/styles/admin-core/content-blocks/_image-text-background.scss

```diff
@@ -36,17 +37,19 @@ $component: "c-block-image-text-background";
 
 	&__button {
 		margin-top: $spacer-xs;
-
-		&-icon {
-			&--right {
-				.c-button__content {
-					flex-direction: row-reverse !important;
-				}
-			}
-		}
 	}
 
 	&__image-wrapper {
+		border: 1px solid red;
```

Comment:
rem

Reply:
Was this border requested? Probably used for testing no?